components/modal/confirm.tsx

```diff
@@ -9,6 +9,7 @@ import { getConfirmLocale } from './locale';
 interface ConfirmDialogProps extends ModalFuncProps {
   afterClose?: () => void;
   close: (...args: any[]) => void;
+  autoFocusBtn: 'ok' | 'cancel';
```

Comment:
Consider https://github.com/ant-design/ant-design/pull/11055 and we can set `autoFocusBtn` to `null|false|undefined` for no focused button.

Reply:
autoFocusBtn => autoFocusButton